usaspending_api/references/migrations/0032_auto_20190926_1308.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-09-26 13:08
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('references', '0031_delete_cgac'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='psc',
+            name='code',
+            field=models.CharField(max_length=4),
+        ),
```

Comment:
I don't see anything wrong with this, but confused why it got added to the migration when it isn't touched on the PSC model.

Reply:
ah yeah, i had to mess with the model a bunch when i was testing stuff, that shouldn't be there at all, good catch